cinema/views.py

```diff
@@ -1 +1,63 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Movie, CinemaHall, Genre, Actor, MovieSession
+from cinema.serializers import (
+    MovieSerializer,
+    CinemaHallSerializer,
+    GenreSerializer,
+    ActorSerializer,
+    MovieSessionSerializer,
+    MovieSessionListSerializer,
+    MovieListSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionRetrieveSerializer
+)
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
```

Comment:
The use of `.all()` before `.prefetch_related()` and `.select_related()` is redundant. You can directly use `CinemaHall.objects.prefetch_related()` or `CinemaHall.objects.select_related()` in the `get_queryset` method to optimize performance.